ext/standard/basic_functions.c

```diff
@@ -2852,6 +2853,7 @@ static const zend_function_entry basic_functions[] = { /* {{{ */
 	PHP_FE(str_pad,															arginfo_str_pad)
 	PHP_FALIAS(chop,				rtrim,									arginfo_rtrim)
 	PHP_FALIAS(strchr,				strstr,									arginfo_strstr)
+	PHP_FALIAS(strrchr,				strrstr,									arginfo_strstr)
```

Comment:
No more alias, please 😢

Reply:
I'm not really sure how to handle it otherwise as this is how ``strchr`` was handled.